snforge_std/src/cheatcodes.cairo

```diff
@@ -22,83 +29,116 @@ fn test_address() -> ContractAddress {
     contract_address_const::<469394814521890341860918960550914>()
 }
 
-fn start_roll(target: CheatTarget, block_number: u64) {
+fn roll(target: CheatTarget, block_number: u64, span: CheatSpan) {
+    validate_cheat_target_and_span(@target, @span);
+
     let mut inputs = array![];
     target.serialize(ref inputs);
+    span.serialize(ref inputs);
     inputs.append(block_number.into());
     cheatcode::<'start_roll'>(inputs.span());
 }
 
+fn start_roll(target: CheatTarget, block_number: u64) {
+    roll(target, block_number, CheatSpan::Indefinite);
+}
+
 fn stop_roll(target: CheatTarget) {
     let mut inputs = array![];
     target.serialize(ref inputs);
     cheatcode::<'stop_roll'>(inputs.span());
 }
 
-fn start_prank(target: CheatTarget, caller_address: ContractAddress) {
+fn prank(target: CheatTarget, caller_address: ContractAddress, span: CheatSpan) {
+    validate_cheat_target_and_span(@target, @span);
+
     let mut inputs = array![];
     target.serialize(ref inputs);
+    span.serialize(ref inputs);
     inputs.append(caller_address.into());
     cheatcode::<'start_prank'>(inputs.span());
```

Comment:
I wonder if we shouldn't change the selectors to correct ones e.g. "prank", "roll" etc.

Reply:
Could do

Reply:
[Rename cheatcodes](https://github.com/foundry-rs/starknet-foundry/pull/1874/commits/df73cbd78e1cf5879bbfaba73a802e4d14be8a66)